documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -14,13 +14,61 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
-. Upgrade the {operator-name}.
+. Change the update channel.
 +
 ifeval::["{build}" == "upstream"]
-See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
 endif::[]
 ifeval::["{build}" == "downstream"]
-See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-upgrading-operators[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
+endif::[]
+
+. Restart the `CatalogSource` pod:
+
+.. In the the {ocp-short} web console, click *Operators* -> *Installed Operators*. Click *Forklift Operator* to view the *Operator details* page.
+
+..  On the *Subscriptions* tab, note the name in the *CatalogSource* field.
+..  From the command line, run the following command:
++
+[source,terminal,subs=attributes+]
+----
+$ {oc} get pod -n openshift-marketplace
+----
+.Example output:
++
+[source,terminal,subs=attributes+]
+----
+NAME                                                              READY   STATUS      RESTARTS   AGE
+56743631b0a246d54b96aaca5d217c1221516ae0359bd69295316c--1-j2vsd   0/1     Completed   0          19h
+7fd3c8520985a7090567f60e97203dfbb6da3a051f1bbc06334e3e--1-n5pvq   0/1     Completed   0          19h
+certified-operators-64bn2                                         1/1     Running     0          34h
+community-operators-97cct                                         1/1     Running     0          15h
+hco-catalogsource-bcf5b                                           1/1     Running     0          4d3h
+marketplace-operator-7744f76566-7h9lg                             1/1     Running     0          8m42s
+ocs-catalogsource-cjdf6                                           1/1     Running     0          4d3h
+redhat-marketplace-8nwq5                                          1/1     Running     0          2d14h
+redhat-operators-mgrrm                                            1/1     Running     0          19h
+----
+
+..  Locate the pod that corresponds to the name in Step 2b.
```

Comment:
Do not refer to a step by its number. 

You could just tell the user to get the pod by name instead of getting the whole list of pods.

Reply:
Or you could get rid of the oc get pod step and just use the oc delete command. The user already has the pod name!